### ISSUE TITLE
[Bug]: Network skip not working on latest game version

### ISSUE THREAD

**Issue by EthanArmbrust** [Lyall/P3RFix#47]
## Issue Summary:
The network screen is not skipped when starting the game on the latest patch. Warnings and logos are still skipped. This worked on the previous game version.

## P3RFix & Game Version Numbers:
Persona 3 Reload: 1.0.173.0 (Xbox)
P3RFix: 1.2.1

## Log Files:
[2024-09-10 00:32:14.894] [P3RFix] [info] ----------
[2024-09-10 00:32:14.894] [P3RFix] [info] P3RFix v1.2.1 loaded.
[2024-09-10 00:32:14.894] [P3RFix] [info] ----------
[2024-09-10 00:32:14.894] [P3RFix] [info] Path to logfile: C:\Program Files\WindowsApps\SEGAofAmericaInc.L0cb6b3aea_1.0.173.0_x64__s751p9cej88mt\P3R\Binaries\WinGDK\P3RFix.log
[2024-09-10 00:32:14.894] [P3RFix] [info] ----------
[2024-09-10 00:32:14.894] [P3RFix] [info] Module Name: P3R.exe
[2024-09-10 00:32:14.894] [P3RFix] [info] Module Path: C:\Program Files\WindowsApps\SEGAofAmericaInc.L0cb6b3aea_1.0.173.0_x64__s751p9cej88mt\P3R\Binaries\WinGDK\
[2024-09-10 00:32:14.894] [P3RFix] [info] Module Address: 0x140000000
[2024-09-10 00:32:14.894] [P3RFix] [info] Module Timestamp: 1724622388
[2024-09-10 00:32:14.894] [P3RFix] [info] ----------
[2024-09-10 00:32:14.894] [P3RFix] [info] Path to config file: C:\Program Files\WindowsApps\SEGAofAmericaInc.L0cb6b3aea_1.0.173.0_x64__s751p9cej88mt\P3R\Binaries\WinGDK\P3RFix.ini
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: iInjectionDelay: 1000ms
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: bCustomResolution: true
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: iCustomResX: 2560
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: iCustomResY: 1440
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: bSkipLogos: true
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: iSkipLogos: 1
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: bUncapMenuFPS: true
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: bPauseOnFocusLoss: false
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: bHUDFix: true
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: bAspectFix: true
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: bFOVFix: true
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: bScreenPercentage: true
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: fScreenPercentage: 100
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: bRenTexResMulti: true
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: fRenTexResUserMulti = 1
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: bAdjustFPSCap: false
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: fFramerateCap: 120
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: bMouseFix: false
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: bIgnoreGamepad: true
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: fMouseMultiplierX: 1
[2024-09-10 00:32:14.894] [P3RFix] [info] Config Parse: fMouseMultiplierY: 1
[2024-09-10 00:32:14.894] [P3RFix] [info] ----------
[2024-09-10 00:32:14.894] [P3RFix] [info] Custom Resolution: fAspectRatio: 1.7777778
[2024-09-10 00:32:14.894] [P3RFix] [info] Custom Resolution: fAspectMultiplier: 1
[2024-09-10 00:32:14.894] [P3RFix] [info] Custom Resolution: fHUDWidth: 2560
[2024-09-10 00:32:14.894] [P3RFix] [info] Custom Resolution: fHUDHeight: 1440
[2024-09-10 00:32:14.894] [P3RFix] [info] Custom Resolution: fHUDWidthOffset: 0
[2024-09-10 00:32:14.894] [P3RFix] [info] Custom Resolution: fHUDHeightOffset: 0
[2024-09-10 00:32:14.894] [P3RFix] [info] ----------
[2024-09-10 00:32:15.930] [P3RFix] [info] Caution Skip: Address is P3R.exe+155f19a
[2024-09-10 00:32:15.930] [P3RFix] [info] Caution Skip: Skipped to network dialog.
[2024-09-10 00:32:15.976] [P3RFix] [info] Network Skip: Address is P3R.exe+1552060
[2024-09-10 00:32:15.976] [P3RFix] [info] Network Skip: Dialog: Address is P3R.exe+126ebd1
[2024-09-10 00:32:15.977] [P3RFix] [info] Network Skip: Skipped over network dialog.
[2024-09-10 00:32:16.020] [P3RFix] [info] Intro Skip: Address is P3R.exe+155fd1b
[2024-09-10 00:32:16.021] [P3RFix] [info] Intro Skip: Skipped intro to title state 4.
[2024-09-10 00:32:16.344] [P3RFix] [info] Custom Resolution: Address is P3R.exe+14667ff0
[2024-09-10 00:32:16.395] [P3RFix] [info] FOV: Address is P3R.exe+2e80e0a
[2024-09-10 00:32:16.458] [P3RFix] [info] Aspect Ratio: Address is P3R.exe+2e80e0f
[2024-09-10 00:32:16.475] [P3RFix] [info] Aspect Ratio Check: Address is P3R.exe+c838b4
[2024-09-10 00:32:16.475] [P3RFix] [info] Aspect Ratio Check: Instruction patched.
[2024-09-10 00:32:16.488] [P3RFix] [info] HUD Constraints: Address is P3R.exe+c95fec
[2024-09-10 00:32:16.506] [P3RFix] [info] Fade Status: Address is P3R.exe+1096f50
[2024-09-10 00:32:16.523] [P3RFix] [info] Fades: Address is P3R.exe+10939ad
[2024-09-10 00:32:16.523] [P3RFix] [info] Fades: Function address is P3R.exe+c9a2a0
[2024-09-10 00:32:16.586] [P3RFix] [info] Screen Percentage: Address is P3R.exe+30caefc
[2024-09-10 00:32:16.655] [P3RFix] [info] Render Texture 2D Resolution: Address is P3R.exe+33ed3f0
[2024-09-10 00:32:16.668] [P3RFix] [info] RT_Capture: Address is P3R.exe+c75cb4
[2024-09-10 00:32:16.686] [P3RFix] [info] Menu FPS Cap: Address is P3R.exe+10ac887
[2024-09-10 00:32:16.686] [P3RFix] [info] Menu FPS Cap: Patched instruction.
[2024-09-10 00:32:17.692] [P3RFix] [info] Window Focus: Set new WndProc.
[2024-09-10 00:32:17.972] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 1.3333334
[2024-09-10 00:32:17.972] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-10 00:32:17.972] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 2560x1440
[2024-09-10 00:32:17.973] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 1.3333334
[2024-09-10 00:32:17.973] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-10 00:32:17.973] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 2560x1440
[2024-09-10 00:32:17.973] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 1.3333334
[2024-09-10 00:32:17.973] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-10 00:32:17.973] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 2560x1440
[2024-09-10 00:32:17.973] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 1.3333334
[2024-09-10 00:32:17.973] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-10 00:32:17.973] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 2560x1440
[2024-09-10 00:32:17.973] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 1.3333334
[2024-09-10 00:32:17.973] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-10 00:32:17.973] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 2560x1440
[2024-09-10 00:32:19.527] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 1.3333334
[2024-09-10 00:32:19.527] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-10 00:32:19.527] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 2560x1440
[2024-09-10 00:32:19.527] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 1.3333334
[2024-09-10 00:32:19.527] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-10 00:32:19.527] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 2560x1440

**Comment by Lyall** [comment]
Should be fixed in [v1.2.3](https://github.com/Lyall/P3RFix/releases)

**Comment by EthanArmbrust** [comment]
Intro skipping doesn't seem to be working at all in v1.2.3 for me anymore. I'm getting warnings and logos as well.  v1.2.2 is working OK.

```
[2024-09-17 01:16:18.516] [P3RFix] [info] ----------
[2024-09-17 01:16:18.516] [P3RFix] [info] P3RFix v1.2.3 loaded.
[2024-09-17 01:16:18.516] [P3RFix] [info] ----------
[2024-09-17 01:16:18.516] [P3RFix] [info] Path to logfile: C:\Program Files\WindowsApps\SEGAofAmericaInc.L0cb6b3aea_1.0.173.0_x64__s751p9cej88mt\P3R\Binaries\WinGDK\P3RFix.log
[2024-09-17 01:16:18.517] [P3RFix] [info] ----------
[2024-09-17 01:16:18.517] [P3RFix] [info] Module Name: P3R.exe
[2024-09-17 01:16:18.517] [P3RFix] [info] Module Path: C:\Program Files\WindowsApps\SEGAofAmericaInc.L0cb6b3aea_1.0.173.0_x64__s751p9cej88mt\P3R\Binaries\WinGDK\
[2024-09-17 01:16:18.517] [P3RFix] [info] Module Address: 0x140000000
[2024-09-17 01:16:18.517] [P3RFix] [info] Module Timestamp: 1724622388
[2024-09-17 01:16:18.517] [P3RFix] [info] ----------
[2024-09-17 01:16:18.517] [P3RFix] [info] Path to config file: C:\Program Files\WindowsApps\SEGAofAmericaInc.L0cb6b3aea_1.0.173.0_x64__s751p9cej88mt\P3R\Binaries\WinGDK\P3RFix.ini
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bCustomResolution: true
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: iCustomResX: 2560
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: iCustomResY: 1440
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bSkipLogos: true
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: iSkipLogos: 1
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bUncapMenuFPS: true
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bPauseOnFocusLoss: false
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bEnableConsole: true
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: sConsoleHotkey: Tilde
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bHUDFix: true
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bAspectFix: true
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bFOVFix: true
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bScreenPercentage: false
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: fScreenPercentage: 100
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bRenTexResMulti: true
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: fRenTexResUserMulti = 1.5
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bAdjustFPSCap: false
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: fFramerateCap: 120
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bMouseFix: false
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: bIgnoreGamepad: true
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: fMouseMultiplierX: 1
[2024-09-17 01:16:18.517] [P3RFix] [info] Config Parse: fMouseMultiplierY: 1
[2024-09-17 01:16:18.517] [P3RFix] [info] ----------
[2024-09-17 01:16:18.517] [P3RFix] [info] ----------
[2024-09-17 01:16:18.517] [P3RFix] [info] Current Resolution: Base Resolution: 2560x1440
[2024-09-17 01:16:18.517] [P3RFix] [info] Current Resolution: Screen Percentage: 100
[2024-09-17 01:16:18.517] [P3RFix] [info] Current Resolution: Scaled Resolution: 2560x1440
[2024-09-17 01:16:18.517] [P3RFix] [info] Current Resolution: fAspectRatio: 1.7777778
[2024-09-17 01:16:18.517] [P3RFix] [info] Current Resolution: fAspectMultiplier: 1
[2024-09-17 01:16:18.517] [P3RFix] [info] Current Resolution: fHUDWidth: 2560
[2024-09-17 01:16:18.517] [P3RFix] [info] Current Resolution: fHUDHeight: 1440
[2024-09-17 01:16:18.517] [P3RFix] [info] Current Resolution: fHUDWidthOffset: 0
[2024-09-17 01:16:18.517] [P3RFix] [info] Current Resolution: fHUDHeightOffset: 0
[2024-09-17 01:16:18.517] [P3RFix] [info] ----------
[2024-09-17 01:16:18.571] [P3RFix] [info] Render Texture 2D Resolution: Address is P3R.exe+33ed3f0
[2024-09-17 01:16:18.583] [P3RFix] [info] RT_Capture: Address is P3R.exe+c75cb4
[2024-09-17 01:16:18.825] [P3RFix] [info] Custom Resolution: Address is P3R.exe+14667ff0
[2024-09-17 01:16:18.871] [P3RFix] [info] Current Resolution: Address is P3R.exe+3419d13
[2024-09-17 01:16:18.890] [P3RFix] [info] ----------
[2024-09-17 01:16:18.890] [P3RFix] [info] Current Resolution: Base Resolution: 1280x720
[2024-09-17 01:16:18.890] [P3RFix] [info] Current Resolution: Screen Percentage: 100
[2024-09-17 01:16:18.890] [P3RFix] [info] Current Resolution: Scaled Resolution: 1280x720
[2024-09-17 01:16:18.890] [P3RFix] [info] Current Resolution: fAspectRatio: 1.7777778
[2024-09-17 01:16:18.890] [P3RFix] [info] Current Resolution: fAspectMultiplier: 1
[2024-09-17 01:16:18.890] [P3RFix] [info] Current Resolution: fHUDWidth: 1280
[2024-09-17 01:16:18.890] [P3RFix] [info] Current Resolution: fHUDHeight: 720
[2024-09-17 01:16:18.890] [P3RFix] [info] Current Resolution: fHUDWidthOffset: 0
[2024-09-17 01:16:18.890] [P3RFix] [info] Current Resolution: fHUDHeightOffset: 0
[2024-09-17 01:16:18.890] [P3RFix] [info] ----------
[2024-09-17 01:16:19.359] [P3RFix] [error] Intro Skip: Pattern scan failed.
[2024-09-17 01:16:19.369] [P3RFix] [info] Offsets: GObjects: Address is P3R.exe+904a9c
[2024-09-17 01:16:19.369] [P3RFix] [info] Offsets: GObjects: Offset: 5898cc0
[2024-09-17 01:16:19.395] [P3RFix] [info] Offsets: AppendString: Address is P3R.exe+1855610
[2024-09-17 01:16:19.395] [P3RFix] [info] Offsets: AppendString: Offset: 1855610
[2024-09-17 01:16:19.417] [P3RFix] [info] Offsets: ProcessEvent: Address is P3R.exe+1a48b80
[2024-09-17 01:16:19.417] [P3RFix] [info] Offsets: ProcessEvent: Offset: 1a48b80
[2024-09-17 01:16:20.032] [P3RFix] [info] ----------
[2024-09-17 01:16:20.032] [P3RFix] [info] Current Resolution: Base Resolution: 2560x1440
[2024-09-17 01:16:20.032] [P3RFix] [info] Current Resolution: Screen Percentage: 100
[2024-09-17 01:16:20.032] [P3RFix] [info] Current Resolution: Scaled Resolution: 2560x1440
[2024-09-17 01:16:20.032] [P3RFix] [info] Current Resolution: fAspectRatio: 1.7777778
[2024-09-17 01:16:20.032] [P3RFix] [info] Current Resolution: fAspectMultiplier: 1
[2024-09-17 01:16:20.032] [P3RFix] [info] Current Resolution: fHUDWidth: 2560
[2024-09-17 01:16:20.032] [P3RFix] [info] Current Resolution: fHUDHeight: 1440
[2024-09-17 01:16:20.032] [P3RFix] [info] Current Resolution: fHUDWidthOffset: 0
[2024-09-17 01:16:20.032] [P3RFix] [info] Current Resolution: fHUDHeightOffset: 0
[2024-09-17 01:16:20.032] [P3RFix] [info] ----------
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 2
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 3840x2160
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 2
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 3840x2160
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 2
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 3840x2160
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 2
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 3840x2160
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 2
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-17 01:16:21.490] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 3840x2160
[2024-09-17 01:16:21.701] [P3RFix] [info] Construct Console: GEngine address = 5ca6a060
[2024-09-17 01:16:21.702] [P3RFix] [info] Construct Console: Console object constructed.
[2024-09-17 01:16:21.703] [P3RFix] [info] Construct Console: Console enabled - access it using key: Tilde.
[2024-09-17 01:16:21.745] [P3RFix] [info] FOV: Address is P3R.exe+2e80e02
[2024-09-17 01:16:21.799] [P3RFix] [info] Aspect Ratio: Address is P3R.exe+2e80e0f
[2024-09-17 01:16:21.812] [P3RFix] [info] Aspect Ratio Check: Address is P3R.exe+c838b4
[2024-09-17 01:16:21.812] [P3RFix] [info] Aspect Ratio Check: Instruction patched.
[2024-09-17 01:16:21.822] [P3RFix] [info] HUD Constraints: Address is P3R.exe+c95fec
[2024-09-17 01:16:21.838] [P3RFix] [info] Fade Status: Address is P3R.exe+1096f50
[2024-09-17 01:16:21.852] [P3RFix] [info] Fades: Address is P3R.exe+10939ad
[2024-09-17 01:16:21.852] [P3RFix] [info] Fades: Function address is P3R.exe+c9a2a0
[2024-09-17 01:16:21.902] [P3RFix] [info] Screen Percentage: Address is P3R.exe+30caefc
[2024-09-17 01:16:21.917] [P3RFix] [info] Menu FPS Cap: Address is P3R.exe+10ac887
[2024-09-17 01:16:21.917] [P3RFix] [info] Menu FPS Cap: Patched instruction.
[2024-09-17 01:16:22.917] [P3RFix] [info] Window Focus: Set new WndProc.
[2024-09-17 01:16:23.015] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 2
[2024-09-17 01:16:23.015] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-17 01:16:23.015] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 3840x2160
[2024-09-17 01:16:23.015] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 2
[2024-09-17 01:16:23.015] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-17 01:16:23.015] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 3840x2160
[2024-09-17 01:16:24.387] [P3RFix] [info] ----------
[2024-09-17 01:16:24.387] [P3RFix] [info] Current Resolution: Base Resolution: 2560x1440
[2024-09-17 01:16:24.387] [P3RFix] [info] Current Resolution: Screen Percentage: 70.6778
[2024-09-17 01:16:24.387] [P3RFix] [info] Current Resolution: Scaled Resolution: 1809.3518x1017.7604
[2024-09-17 01:16:24.387] [P3RFix] [info] Current Resolution: fAspectRatio: 1.7777778
[2024-09-17 01:16:24.387] [P3RFix] [info] Current Resolution: fAspectMultiplier: 1
[2024-09-17 01:16:24.387] [P3RFix] [info] Current Resolution: fHUDWidth: 2560
[2024-09-17 01:16:24.387] [P3RFix] [info] Current Resolution: fHUDHeight: 1440
[2024-09-17 01:16:24.387] [P3RFix] [info] Current Resolution: fHUDWidthOffset: 0
[2024-09-17 01:16:24.387] [P3RFix] [info] Current Resolution: fHUDHeightOffset: 0
[2024-09-17 01:16:24.387] [P3RFix] [info] ----------
[2024-09-17 01:16:53.038] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 1.4135561
[2024-09-17 01:16:53.038] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-17 01:16:53.038] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 2714x1526
[2024-09-17 01:16:53.038] [P3RFix] [info] Render Texture 2D Resolution: fRenTexResMulti = 1.4135561
[2024-09-17 01:16:53.038] [P3RFix] [info] Render Texture 2D Resolution: Old render texture resolution = 1920x1080
[2024-09-17 01:16:53.038] [P3RFix] [info] Render Texture 2D Resolution: New render texture resolution = 2714x1526
```

**Comment by Lyall** [comment]
Ah it looks like some/all the intro skip patterns are failing on the Xbox/MS Store version. Can you dump the latest game exe and upload it?

You can use [UWPDumper](https://github.com/Wunkolo/UWPDumper/releases/download/latest/UWPDumper.zip) to dump the game exe after launching it. Then if you upload it somewhere I can check the patterns against the exe.

**Comment by EthanArmbrust** [comment]
https://drive.google.com/file/d/1A_xMPobOaqzpGvv2AXth2XgyIrH0MeBZ/view?usp=sharing

**Comment by nintendork07** [comment]
can also confirm, the skip intro videos doesn't work for me on the latest xbox game pass version either. version 1.0.173.0 with the aegis expansion installed

**Comment by Lyall** [comment]
![image](https://github.com/user-attachments/assets/2b68a8fe-1b0a-4bfa-974c-505fcecd638c)

Sorry for taking so long to get this done. I've updated the patterns to work on both the MS Store and Steam versions. If someone who has access to the MS Store/Xbox version of the game can verify this works, I'll put out a new release.

[P3RFix_v1.2.4_Xbox_test.zip](https://github.com/user-attachments/files/17327974/P3RFix_v1.2.4_Xbox_test.zip)

**Comment by Borhmian** [comment]
> ![image](https://private-user-images.githubusercontent.com/695941/375371850-2b68a8fe-1b0a-4bfa-974c-505fcecd638c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4NDI4ODQsIm5iZiI6MTcyODg0MjU4NCwicGF0aCI6Ii82OTU5NDEvMzc1MzcxODUwLTJiNjhhOGZlLTFiMGEtNGJmYS05NzRjLTUwNWZjZWNkNjM4Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxM1QxODAzMDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yYWE0NmNlODYzYzZiNTk2ZjUyYTlkOWRkODdkMjkxMGYxMmJlMjk0MjA5Y2E3OGRhYTE1MDM0NzYzMThkZDRjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.oE2foLBDPHNad8iaGMax86XUVS43YLvTrFc8k5Tfvas)
> 
> Sorry for taking so long to get this done. I've updated the patterns to work on both the MS Store and Steam versions. If someone who has access to the MS Store/Xbox version of the game can verify this works, I'll put out a new release.
> 
> [P3RFix_v1.2.4_Xbox_test.zip](https://github.com/user-attachments/files/17327974/P3RFix_v1.2.4_Xbox_test.zip)

I just tested it and it seems to be working with the GamePass version of P3R. Thank you so much!

**Comment by Lyall** [comment]
Fixed in v1.2.4
